bpfd/src/config.rs

```diff
@@ -0,0 +1,215 @@
+// SPDX-License-Identifier: (MIT OR Apache-2.0)
```

Comment:
Did this change? Not sure why it's not just coming up as a rename? Or maybe a rename and then some changes